test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -730,6 +730,44 @@ def test_shufflenet_v2_dynamic_axes(self):
                       dynamic_axes={"input_images": {0: "batch_size"}, "output": {0: "batch_size"}},
                       rtol=1e-3, atol=1e-5)
 
+    @disableScriptTest()
+    def test_mobilenet_v3(self):
+        model = torchvision.models.quantization.mobilenet_v3_large(pretrained=False)
+        dummy_input = torch.randn(1, 3, 224, 224, requires_grad=True)
+        self.run_test(model, (dummy_input,))
+
+    @unittest.skip("Fixed in PyTorch master. RuntimeError: Error(s) in loading state_dict for QuantizableMobileNetV3")
```

Comment:
Unfortunately loading pretrained mobilenet fails under current `onnx_ms_1` branch. The fix commit is currently in master. Yet this PR needs to be opened against onnx_ms_1, since it depends on the previous quantization PR.

Reply:
Can you please add the commit hash or PR # here that has the fix in this string?

Reply:
Possibly #71438, but I can't say for sure which one. The error goes away when I rebase on top of latest master.